components/pedigree/resources/src/main/resources/pedigree/view/workspace.js

```diff
@@ -139,17 +140,17 @@ define([
             // set display:block
             svgText = svgText.replace(/(<svg[^<>]+style=")/g, "$1display:block; ");
```

Comment:
@meatcar just out of curiosity, is it looks like the case of chaining .replace() which is inefficient (like it was in my code for variants), or we can not optimize here (like template evaluating)?